docs/src/docs/arc42/cross-cutting/operational-concepts.adoc

```diff
@@ -6,18 +6,26 @@
 The IRS can be configured using two mechanisms:
 
 ==== application.yml
-If you build the IRS yourself, you can modify the application.yml config that is shipped with the IRS. This file contains all possible config entries for the application.
-Once the Docker image has been built, these values can only be overwritten using the Spring external config mechanism (see https://docs.spring.io/spring-boot/docs/2.1.9.RELEASE/reference/html/boot-features-external-config.html), e.g. by mounting a config file in the right path or using environment variables.
+If you build the IRS yourself, you can modify the application.yml config that is shipped with the IRS.
+This file contains all possible config entries for the application.
+Once the Docker image has been built, these values can only be overwritten using the
+https://docs.spring.io/spring-boot/docs/3.1.9/reference/htmlsingle/#features.external-config[Spring external config mechanism],
```

Comment:
O would not recommend to use versions in this abstract here

Reply:
Hi @mkanal, I stumbeled over this too. The path changed between versions in the past - not only the version number, which is why we cannot update automatically using resource filtering. We would have to remove the link completely if we do not want to include a version here, which is not a good idea IMHO because then this information would be hard to find for people that are not so familiar with Spring Boot yet. As the mechanism of external config is pretty stable over the versions I think it is better to have a link to an old version here (if we forget to update) than to remove it completely.

Reply:
IMHO a generic info that we are using spring boot configuration is enough we do not have to point to the documentation. Spring Boot is well known and any developer could find the corresponding documentation istnt it the case?

Reply:
For my part and currently contributing colleagues this is probably true.  But maybe not for future contributors. As the relevant section of the documentation can be found by searching for "spring boot externalized configuration" in search engines it may be ok to remove the link.

Reply:
link removed